src/esp/assets/ResourceManager.cpp

```diff
@@ -47,6 +47,8 @@
 
 #include "esp/nav/PathFinder.h"
 
+#include "deps/v-hacd/src/VHACD_Lib/public/VHACD.h"
```

Comment:
Hm, since the `public/` include directory is now in the include path, shouldn't this be 

```suggestion
#include "VHACD.h"
```

instead? I'm not a fan of hardcoding the full paths this way :)